street_network/src/lanes/mod.rs

```diff
@@ -76,17 +83,22 @@ impl LaneType {
             LaneType::Construction => false,
             LaneType::LightRail => true,
             LaneType::Buffer(_) => false,
+            LaneType::Footway => true,
+            LaneType::SharedUse => true,
         }
     }
 
     pub fn is_walkable(self) -> bool {
-        self == LaneType::Sidewalk || self == LaneType::Shoulder
+        matches!(
+            self,
+            LaneType::Sidewalk | LaneType::Shoulder | LaneType::Footway | LaneType::SharedUse
+        )
     }
 
     pub fn describe(self) -> &'static str {
         match self {
             LaneType::Driving => "a general-purpose driving lane",
-            LaneType::Biking => "a protected bike lane",
```

Comment:
"protected" never ever made sense, I erroneously wrote this long ago.

Also this is a localization issue... "cycling lane" or "cycle lane" would fit better in the UK. But anyway.